.github/workflows/benchmark-pg_search.yml

```diff
@@ -94,14 +95,76 @@ jobs:
         working-directory: benchmarks/
         run: cargo run -- --url postgresql://localhost:288${{ matrix.pg_version }}/postgres --rows ${{ matrix.num_rows }} --type pg_search --output csv
 
-      - name: Send Slack Notification
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_BENCHMARKS_CHANNEL_WEBHOOK_URL }}
-        working-directory: benchmarks/
+      - name: Derive Short Commit
+        id: commit_info
         run: |
-          for file in results_pg_search_{test_info,index_creation,benchmark_results,postgres_settings}.csv; do
-            curl -F "file=@$file" \
-              -F "channels=${{ secrets.SLACK_BENCHMARKS_CHANNEL_ID }}" \
-              -H "Authorization: Bearer ${{ secrets.SLACK_OAUTH_TOKEN }}" \
-              https://slack.com/api/files.upload
-          done
+          short_commit=$(echo "${GITHUB_SHA}" | cut -c1-7)
+          echo "short_commit=$short_commit" >> $GITHUB_OUTPUT
+
+      - name: Upload Benchmark Parameters CSV to Slack
+        uses: slackapi/slack-github-action@v2.0.0
+        with:
+          method: files.uploadV2
+          token: ${{ secrets.SLACK_OAUTH_TOKEN }}
+          payload: |
+            channel_id: ${{ secrets.SLACK_BENCHMARKS_CHANNEL_ID }}
+            initial_comment: |
+              *Community Benchmarks Parameters*
+              <${{ github.server_url }}/${{ github.repository }}/commit/${{ github.sha }}|${{ steps.commit_info.outputs.short_commit }}> | <${{ steps.artifact-logs.outputs.artifact-url }} | Download Logs>
+            file: "benchmarks/test_info.csv"
```

Comment:
The file name generated by the Rust code has format `results_{}_test_info.csv`:

https://github.com/paradedb/paradedb/blob/7958d89bc025709460748e864d200eba5a34c076/benchmarks/src/main.rs#L61